docs/reference/indices/split-index.asciidoc

```diff
@@ -47,6 +49,27 @@ Splitting works as follows:
 * Finally, it recovers the target index as though it were a closed index which
   had just been re-opened.
 
+[float]
+=== Why doesn't Elasticsearch support incremental resharding?
+
+Going from `N` shards to `N+1` shards, aka. incremental resharding is indeed a
+feature that is supported by many key-value stores, which leverage the fact that
+consistent hashing only requires `1/N`-th of the keys to be relocated. However
+Elasticsearch's unit of storage, shards, are Lucene indices. Unfortunately,
```

Comment:
I think we can leave `Unfortunately` out. Maybe something like `Because of their search-optimised structure`, or maybe just nothing at all.

Reply:
I like "because of their search-optimized structure" because it explains why it isn't a thing for us.